spec/Section 6 -- Execution.md

```diff
@@ -20,6 +20,10 @@ Given this information, the result of {ExecuteRequest(schema, document,
 operationName, variableValues, initialValue)} produces the response, to be
 formatted according to the Response section below.
 
+A GraphQL service must ignore unrecognized information in a request. This allows
```

Comment:
This seems a little broad; arguably something inside the document (an unknown directive, a new syntactic symbol, fragment arguments) is in the request and could be unrecognized, and those should result in validation/parse errors rather than being ignored. Maybe we can use “additional” instead, such as “must ignore any additional unrecognised information” - that implies it’s outside of what’s already specified here?

Reply:
Makes sense to me 👍 . I know @leebyron wanted to make some wording changes, I'll defer to his decision.